Misc/NEWS.d/next/Security/2019-09-25-13-21-09.bpo-38243.1pfz24.rst

```diff
@@ -0,0 +1,2 @@
+Escape the server title of :class:`DocXMLRPCServer` when rendering the
```

Comment:
Sphinx cannot find the class: you have to specify the fully qualified name:

:class:`xmlrpc.server.DocXMLRPCServer`

Reply:
@vstinner Oh.. got it.